Makefile

```diff
@@ -66,5 +66,12 @@ pr:
 	OPAMYES=1 opam publish submit $(NAME).$(VERSION) && rm -rf $(NAME).$(VERSION)
 
 ppx:
+	./configure
```

Comment:
So that configure is called when I make. Without this, I have to do `./configure` manually when the config does not exit. I put this for convenience given that the configure phase is quick…